testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -321,6 +322,7 @@ public void testUpdateFullAcidWithOriginalFilesPrestoInserting(boolean isPartiti
     }
 
     @Test(groups = {STORAGE_FORMATS, HIVE_TRANSACTIONAL}, dataProvider = "partitioningAndBucketingTypeDataProvider", timeOut = TEST_TIMEOUT)
+    @Flaky(issue = ERROR_COMMITTING_WRITE_TO_HIVE_ISSUE, match = ERROR_COMMITTING_WRITE_TO_HIVE_MATCH)
```

Comment:
Do you happen to know why `io.trino.tests.utils.QueryExecutors#connectToPresto` didn't work?

Reply:
Aparently it does not work for `QueryExecutor` obtained with `defaultQueryExecutor()`